frontend/wallet/src/render/components/TransactionCell.re

```diff
@@ -271,51 +266,58 @@ module InfoSection = {
   };
 };
 
-// TODO: As suggested in https://github.com/CodaProtocol/coda/pull/2260 , we
-// should probably switch to CSS Grid
-//
-// We can represent one of these cells as a row in an HTML table in the
-// following manner:
-//
-// ┌────────┬────┬───────────┬─────────────────────────
-// │ Sender │ -> │ Recipient │ Info (including amount)
-//
-// The final column is info and amount in order to more easily support the
-// expanded view of the staking reward
+// These cells are returned as a fragment so that the parent container can
+// use a consistent grid layout to keep everything lined up.
 
 [@react.component]
 let make = (~transaction: Transaction.t, ~myWallets: list(PublicKey.t)) => {
-  let (expanded, setExpanded) = React.useState(() => false);
-
   let viewModel = ViewModel.ofTransaction(transaction, ~myWallets);
 
-  <tr
-    onClick={_e =>
-      switch (viewModel.info) {
-      | StakingReward(_, _) => setExpanded(expanded => !expanded)
-      | _ => ()
-      }
-    }>
-    <td className=Css.(style([verticalAlign(`baseline)]))>
+  <>
+    <span>
       <ActorName value={viewModel.sender} />
-    </td>
-    <td className=Css.(style([verticalAlign(`baseline)]))>
-      {ReasonReact.string(
-         switch (viewModel.action) {
-         | Transfer => "->"
-         | Pending => ">>"
-         | Failed => "x"
-         },
-       )}
-    </td>
-    <td className=Css.(style([verticalAlign(`baseline)]))>
-      <ActorName value={viewModel.recipient} />
-    </td>
-    <td
+      <div>
+        {ReasonReact.string(
+           switch (viewModel.action) {
+           | Transfer => " -> "
+           | Pending => " ... "
+           | Failed => " x "
+           },
+         )}
+        <ActorName value={viewModel.recipient} />
+      </div>
+    </span>
+    <span
       className=Css.(
         style([verticalAlign(`baseline), width(`percent(100.))])
       )>
-      <InfoSection expanded viewModel />
-    </td>
-  </tr>;
+      <InfoSection viewModel />
+    </span>
+    <span className=Css.(style([justifySelf(`flexEnd)]))>
+      <div
+        className=Css.(style([display(`flex), justifyContent(`flexEnd)]))>
+        {switch (viewModel.info) {
```

Comment:
This logic will be refactored into a seperate component when I redo the style stuff.